queue/queue.py

```diff
@@ -8,10 +8,15 @@ def __init__(self):
     self.storage = LinkedList()
 
   def enqueue(self, item):
-    pass
+    self.storage.add_to_tail(item)
   
   def dequeue(self):
-    pass
+    return self.storage.remove_head()
```

Comment:
Above this line, you'll need to check if `self.size` is greater than 0, then decrement it.